tests/e2e/simpleevmtest.go

```diff
@@ -0,0 +1,463 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"google.golang.org/grpc"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	"github.com/oasisprotocol/oasis-core/go/common/logging"
+
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/client"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/crypto/signature"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/testing"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
+
+	"github.com/oasisprotocol/oasis-sdk/tests/e2e/txgen"
+)
+
+// The evmCreateTx type must match the CreateTx type from the evm module types
+// in runtime-sdk/src/modules/evm/types.rs.
+type evmCreateTx struct {
```

Comment:
Could you move these to the client-sdk, e.g. to `client-sdk/go/modules/evm/types.go`.